### PR TITLE
Docs cleanup

### PR DESCRIPTION
**Pull request by CITguy**
### JIRA
* SURF-1619 (partial; no component renaming)
* SURF-1620 (partial; element description only)
* SURF-1621 (partial; test page banner, test breadcrumbs)

### LGTM's
- [ ] Dev LGTM
- [ ] Zoom LGTM

### CODE REVIEW

**Comment by HelixUI** [comment]
Deploy preview for *helix-ui* processing.

Building with commit e75aa1a27213f296531377da155e8b0b2002a251

https://app.netlify.com/sites/helix-ui/deploys/5cba389772a49d0008942893

File: docs/docs.less [hunk 1]
```diff
@@ -248,35 +288,78 @@ dl:not(.hxList) {
...
+}
 
 /* ----- Test Page Styling ----- */
 .test-content {
+  box-shadow: inset 0 0 0 0.5rem @yellow-900;
+  overflow: visible !important; // allow banner to be sticky
+
   section {
     > * {
+      // FIXME: breaks positioned elements
```
Comment (CITguy) [inline docs/docs.less]: - [ ] FIXME: Adds top margin to positioned elements (shifting them down, too far)

**Comment by CITguy** [comment]
2019-04-26: This PR will be broken up into smaller parts.

**Comment by CITguy** [comment]
Closing. All changes handled in separate PRs.

**Pull request closed without merging**